Userland/Libraries/LibGfx/Vector2.h

```diff
@@ -50,6 +50,11 @@ class Vector2 final {
         return Vector2(m_x - other.m_x, m_y - other.m_y);
     }
 
+    constexpr Vector2 operator-() const
```

Comment:
nitpick: `LibGFX` -> `LibGfx`

Reply:
Good spot! Should've let go of the SHIFT key!

Fixed